### PR TITLE
Add videojs-markers@0.7.0 w/ git auto-update

### LINKED ISSUE #9816: [Request] Add videojs-markers
**Library name:** videojs-markers
**Git repository url:** https://github.com/spchuang/videojs-markers
**License(s):** MIT
**Official homepage:** http://sampingchuang.com/videojs-markers

In this issue, we are going to host a new web front-end library on cdnjs, so that web developer can easily use it on our free CDN without additional download/upload process 🚀 .

If you'll like to work locally, I'll suggest you start with sparseCheckout: https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md, since there are too many files in this repo, without sparseCheckout, the whole process will be super slow, using git command line can learn more skills and experience, if you don't like that, instead, you can also just do it on GitHub, it's more convenient, but it may be a little bit harder to fix problems on GitHub gui.

Here are the docs about how to add a library:
 - https://github.com/cdnjs/cdnjs#adding-a-new-or-updating-an-existing-library
 - https://github.com/cdnjs/cdnjs/blob/master/.github/CONTRIBUTING.md#d-adding-a-new-library

We'll also ask to setup auto-update config so that the added libraries can be updated to the latest version easily:
 - https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

For beginners who is interested in join and help add this library, please feel free to tell me what kind of help you'll need, we'll help you send the valid pull request, thanks 😄 .

### PR DESCRIPTION
**Pull request by abdulamite**
close #9816, cc @spchuang

Pull request for issue: #9816

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/spchuang/videojs-markers
 * Official website (optional, not the repository): http://www.sampingchuang.com/videojs-markers
 * License and its reference:  [MIT](https://github.com/dobtco/formbuilder/blob/gh-pages/LICENSE.md)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 7
   - Count of stars: 111
   - Count of forks: 60

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by abdulamite** [comment]
So I went through and made the fixes, how will this PR be updated?

**Comment by pvnr0082t** [comment]
@abdulamite Then, you commit your changes on your side and push it to `videojs-markers` branch.

**Comment by abdulamite** [comment]
I have added the homepage!

File: ajax/libs/videojs-markers/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "keywords": [
+    "videojs",
+    "tags",
+    "marker",
+    "video",
+    "player"
+  ],
+  "homepage": "http://www.sampingchuang.com/videojs-markers",
+  "author": "spchuang",
```
Comment (hare1039) [inline ajax/libs/videojs-markers/package.json]: I think we can add more information about the author.
e.g.
```js
  "author": {
    "name": "spchuang",
    "url": "https://github.com/spchuang"
  }
```
or `"author": "spchuang (https://github.com/spchuang)"`

Comment (abdulamite) [inline ajax/libs/videojs-markers/package.json]: on it!

**Pull request merged**